### ISSUE TITLE
Use  implicit model

### ISSUE THREAD

**Issue by derrick-xwp** [NVlabs/nvdiffrec#41]
Hi

the mesh seems be explicitly learned, how to modify the config for implicit learning?

Thanks

**Comment by jmunkberg** [comment]
I'm not sure I understand the question, can you provide more details?

**Comment by LZleejean** [comment]
I think he expect that the sdf value is predicted by a neural network.

**Comment by derrick-xwp** [comment]
Yes， using  MLP to predict SDF. Is the code only a partial implementation of the original paper?

**Comment by jmunkberg** [comment]
Yes, we didn't include the DTU experiments from our supplemental material with the MLP prediction of the SDF in the public code release. 

Some additional details in this issue: https://github.com/NVlabs/nvdiffrec/issues/9

**Comment by Tom94God** [comment]
Could you open the code with the MLP prediction of the SDF? It seems that the whole framework have to be changed when using MLP to predict SDF.

**Comment by jmunkberg** [comment]
No, we do not plan to release that part of the code unfortunately. 

Implementation details are included in Section 8.5 and 9.2 of our paper. 
https://nvlabs.github.io/nvdiffrec/assets/paper.pdf
It is mostly the dmtet.py file that needs to be modified.

We use the same MLP as in Differentiable Volumetric Rendering:
Learning Implicit 3D Representations without 3D Supervision, with code available here: https://github.com/autonomousvision/differentiable_volumetric_rendering
